diff -urN old/gawk-4.0.1/Makefile.am new/gawk-4.0.1/Makefile.am
--- old/gawk-4.0.1/Makefile.am	2012-03-28 19:47:53 +0000
+++ new/gawk-4.0.1/Makefile.am	2021-03-28 20:35:24 +0000
@@ -120,7 +120,7 @@
 dgawk_SOURCES = $(base_sources) eval_d.c profile.c cmd.h command.y debug.c
 
 # Get extra libs as needed, Automake will supply LIBINTL and SOCKET_LIBS.
-LDADD = $(LIBSIGSEGV) $(LIBINTL) $(SOCKET_LIBS)
+LDADD = $(LIBSIGSEGV) $(LTLIBINTL) $(SOCKET_LIBS)
 dgawk_LDADD = $(LDADD) @LIBREADLINE@
 
 # Directory for gawk's data files. Automake supplies datadir.
diff -urN old/gawk-4.0.1/configure.ac new/gawk-4.0.1/configure.ac
--- old/gawk-4.0.1/configure.ac	2012-03-28 20:01:23 +0000
+++ new/gawk-4.0.1/configure.ac	2021-03-28 20:33:55 +0000
@@ -40,7 +40,7 @@
 fi
 
 AC_PREREQ(2.68)
-AM_INIT_AUTOMAKE([1.11 dist-xz])
+AM_INIT_AUTOMAKE([1.11 dist-xz silent-rules])
 
 AC_CONFIG_MACRO_DIR([m4])
 
@@ -154,8 +154,8 @@
 AC_TYPE_SIGNAL
 AC_SIZE_T
 AC_TYPE_GETGROUPS
-gl_AC_TYPE_LONG_LONG
-gl_AC_TYPE_UNSIGNED_LONG_LONG
+AC_TYPE_LONG_LONG_INT
+AC_TYPE_UNSIGNED_LONG_LONG_INT
 gl_AC_TYPE_INTMAX_T
 gl_AC_TYPE_UINTMAX_T
 AC_CHECK_TYPE(ssize_t, int)
diff -urN old/gawk-4.0.1/m4/intmax_t.m4 new/gawk-4.0.1/m4/intmax_t.m4
--- old/gawk-4.0.1/m4/intmax_t.m4	2010-10-31 10:25:52 +0000
+++ new/gawk-4.0.1/m4/intmax_t.m4	2021-03-28 20:37:01 +0000
@@ -13,7 +13,7 @@
   AC_REQUIRE([gl_AC_HEADER_INTTYPES_H])
   AC_REQUIRE([gl_AC_HEADER_STDINT_H])
   if test $gl_cv_header_inttypes_h = no && test $gl_cv_header_stdint_h = no; then
-    AC_REQUIRE([gl_AC_TYPE_LONG_LONG])
+    AC_REQUIRE([AC_TYPE_LONG_LONG_INT])
     test $ac_cv_type_long_long = yes \
       && ac_type='long long' \
       || ac_type='long'
diff -urN old/gawk-4.0.1/test/Gentests new/gawk-4.0.1/test/Gentests
--- old/gawk-4.0.1/test/Gentests	2012-03-20 18:56:40 +0000
+++ new/gawk-4.0.1/test/Gentests	2021-03-29 20:07:04 +0000
@@ -92,7 +92,8 @@
 	}
 
 	printf "\t@echo %s\n", x
-	printf "\t@AWKPATH=$(srcdir) $(AWK) -f $@.awk %s >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@\n", s
+	printf "\t@AWKPATH=$(srcdir) $(AWK) -f $@.awk %s >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@\n", s
+	printf "\t@sed \"s|awk$(EXEEXT)|awk|g\" < __$@ > _$@ && rm -f __$@\n"
 	printf "\t@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@\n\n"
 }
 
diff -urN old/gawk-4.0.1/test/Makefile.am new/gawk-4.0.1/test/Makefile.am
--- old/gawk-4.0.1/test/Makefile.am	2012-03-28 19:47:54 +0000
+++ new/gawk-4.0.1/test/Makefile.am	2021-03-31 21:16:38 +0000
@@ -869,8 +869,8 @@
 GENTESTS_UNUSED = Makefile.in gtlnbufv.awk printfloat.awk
 
 CMP = cmp
-AWKPROG = ../gawk$(EXEEXT)
-PGAWKPROG = ../pgawk$(EXEEXT)
+AWKPROG = ../gawk
+PGAWKPROG = ../pgawk
 
 # This business forces the locale to be C for running the tests,
 # unless we override it to something else for testing.
@@ -950,7 +950,8 @@
 lc_num1:
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 
@@ -983,11 +984,12 @@
 	.)	: ;; \
 	*)	cp $(srcdir)/argarray.in . ;; \
 	esac
-	@TEST=test echo just a test | $(AWK) -f $(srcdir)/argarray.awk ./argarray.in - >_$@
+	@TEST=test echo just a test | $(AWK) -f $(srcdir)/argarray.awk ./argarray.in - >__$@
 	@case $(srcdir) in \
 	.)	: ;; \
 	*)	rm -f ./argarray.in ;; \
 	esac
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 regtest::
@@ -1008,38 +1010,45 @@
 
 compare::
 	@echo $@
-	@$(AWK) -f $(srcdir)/compare.awk 0 1 $(srcdir)/compare.in >_$@
+	@$(AWK) -f $(srcdir)/compare.awk 0 1 $(srcdir)/compare.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 inftest::
 	@echo $@
 	@echo This test is very machine specific...
-	@$(AWK) -f $(srcdir)/inftest.awk | sed "s/inf/Inf/g" >_$@
+	@$(AWK) -f $(srcdir)/inftest.awk | sed "s/inf/Inf/g" >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getline2::
 	@echo $@
-	@$(AWK) -f $(srcdir)/getline2.awk $(srcdir)/getline2.awk $(srcdir)/getline2.awk >_$@
+	@$(AWK) -f $(srcdir)/getline2.awk $(srcdir)/getline2.awk $(srcdir)/getline2.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 awkpath::
 	@echo $@
-	@AWKPATH="$(srcdir)$(PATH_SEPARATOR)$(srcdir)/lib" $(AWK) -f awkpath.awk >_$@
+	@AWKPATH="$(srcdir)$(PATH_SEPARATOR)$(srcdir)/lib" $(AWK) -f awkpath.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 argtest::
 	@echo $@
-	@$(AWK) -f $(srcdir)/argtest.awk -x -y abc >_$@
+	@$(AWK) -f $(srcdir)/argtest.awk -x -y abc >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 badargs::
 	@echo $@
-	@-$(AWK) -f 2>&1 | grep -v patchlevel >_$@
+	@-$(AWK) -f 2>&1 | grep -v patchlevel >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nonl::
 	@echo $@
-	@-AWKPATH=$(srcdir) $(AWK) --lint -f nonl.awk /dev/null >_$@ 2>&1
+	@-AWKPATH=$(srcdir) $(AWK) --lint -f nonl.awk /dev/null >__$@ 2>&1
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 strftime::
@@ -1053,23 +1062,27 @@
 
 litoct::
 	@echo $@
-	@echo ab | $(AWK) --traditional -f $(srcdir)/litoct.awk >_$@
+	@echo ab | $(AWK) --traditional -f $(srcdir)/litoct.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 devfd::
 	@echo $@
-	@$(AWK) 1 /dev/fd/4 /dev/fd/5 4<$(srcdir)/devfd.in4 5<$(srcdir)/devfd.in5 >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@$(AWK) 1 /dev/fd/4 /dev/fd/5 4<$(srcdir)/devfd.in4 5<$(srcdir)/devfd.in5 >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fflush::
 	@echo $@
-	@$(srcdir)/fflush.sh >_$@
+	@$(srcdir)/fflush.sh >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 tweakfld::
 	@echo $@
-	@$(AWK) -f $(srcdir)/tweakfld.awk $(srcdir)/tweakfld.in >_$@
+	@$(AWK) -f $(srcdir)/tweakfld.awk $(srcdir)/tweakfld.in >__$@
 	@rm -f errors.cleanup
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mmap8k::
@@ -1079,7 +1092,8 @@
 
 tradanch::
 	@echo $@
-	@$(AWK) --traditional -f $(srcdir)/tradanch.awk $(srcdir)/tradanch.in >_$@
+	@$(AWK) --traditional -f $(srcdir)/tradanch.awk $(srcdir)/tradanch.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 # AIX /bin/sh exec's the last command in a list, therefore issue a ":"
@@ -1091,15 +1105,17 @@
 
 strftlng::
 	@echo $@
-	@TZ=UTC; export TZ; $(AWK) -f $(srcdir)/strftlng.awk >_$@
+	@TZ=UTC; export TZ; $(AWK) -f $(srcdir)/strftlng.awk >__$@
 	@if $(CMP) $(srcdir)/strftlng.ok _$@ >/dev/null 2>&1 ; then : ; else \
-	TZ=UTC0; export TZ; $(AWK) -f $(srcdir)/strftlng.awk >_$@ ; \
+	TZ=UTC0; export TZ; $(AWK) -f $(srcdir)/strftlng.awk >__$@ ; \
 	fi
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nors::
 	@echo $@
-	@echo A B C D E | tr -d '\12\15' | $(AWK) '{ print $$NF }' - $(srcdir)/nors.in > _$@
+	@echo A B C D E | tr -d '\12\15' | $(AWK) '{ print $$NF }' - $(srcdir)/nors.in > __$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fmtspcl.ok: fmtspcl.tok Makefile
@@ -1107,23 +1123,27 @@
 
 fmtspcl: fmtspcl.ok
 	@echo fmtspcl
-	@$(AWK) -f $(srcdir)/fmtspcl.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) -f $(srcdir)/fmtspcl.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $@.ok _$@ && rm -f _$@
-
+	
 reint::
 	@echo $@
-	@$(AWK) --re-interval -f $(srcdir)/reint.awk $(srcdir)/reint.in >_$@
+	@$(AWK) --re-interval -f $(srcdir)/reint.awk $(srcdir)/reint.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 pipeio1::
 	@echo $@
-	@$(AWK) -f $(srcdir)/pipeio1.awk >_$@
+	@$(AWK) -f $(srcdir)/pipeio1.awk >__$@
 	@rm -f test1 test2
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 pipeio2::
 	@echo $@
-	@$(AWK) -v SRCDIR=$(srcdir) -f $(srcdir)/pipeio2.awk >_$@
+	@$(AWK) -v SRCDIR=$(srcdir) -f $(srcdir)/pipeio2.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 clobber::
@@ -1134,7 +1154,8 @@
 
 arynocls::
 	@echo $@
-	@-AWKPATH=$(srcdir) $(AWK) -v INPUT=$(srcdir)/arynocls.in -f arynocls.awk >_$@
+	@-AWKPATH=$(srcdir) $(AWK) -v INPUT=$(srcdir)/arynocls.in -f arynocls.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getlnbuf::
@@ -1168,27 +1189,32 @@
 
 redfilnm::
 	@echo $@
-	@$(AWK) -f $(srcdir)/redfilnm.awk srcdir=$(srcdir) $(srcdir)/redfilnm.in >_$@
+	@$(AWK) -f $(srcdir)/redfilnm.awk srcdir=$(srcdir) $(srcdir)/redfilnm.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 leaddig::
 	@echo $@
-	@$(AWK) -v x=2E  -f $(srcdir)/leaddig.awk >_$@
+	@$(AWK) -v x=2E  -f $(srcdir)/leaddig.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst3::
 	@echo $@
-	@$(AWK) --re-interval -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@
+	@$(AWK) --re-interval -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 space::
 	@echo $@
-	@$(AWK) -f ' ' $(srcdir)/space.awk >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) -f ' ' $(srcdir)/space.awk >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 printf0::
 	@echo $@
-	@$(AWK) --posix -f $(srcdir)/$@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) --posix -f $(srcdir)/$@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsnulbig::
@@ -1196,7 +1222,8 @@
 	@ : Suppose that block size for pipe is at most 128kB:
 	@$(AWK) 'BEGIN { for (i = 1; i <= 128*64+1; i++) print "abcdefgh123456\n" }' 2>&1 | \
 	$(AWK) 'BEGIN { RS = ""; ORS = "\n\n" }; { print }' 2>&1 | \
-	$(AWK) '/^[^a]/; END{ print NR }' >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	$(AWK) '/^[^a]/; END{ print NR }' >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsnulbig2::
@@ -1204,92 +1231,108 @@
 	@$(AWK) 'BEGIN { ORS = ""; n = "\n"; for (i = 1; i <= 10; i++) n = (n n); \
 		for (i = 1; i <= 128; i++) print n; print "abc\n" }' 2>&1 | \
 		$(AWK) 'BEGIN { RS = ""; ORS = "\n\n" };{ print }' 2>&1 | \
-		$(AWK) '/^[^a]/; END { print NR }' >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+		$(AWK) '/^[^a]/; END { print NR }' >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 wideidx::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 wideidx2::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 widesub::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 widesub2::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 widesub3::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 widesub4::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ignrcas2::
 	@echo $@
 	@GAWKLOCALE=en_US ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 subamp::
 	@echo $@
 	@GAWKLOCALE=en_US.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 # This test makes sure gawk exits with a zero code.
 # Thus, unconditionally generate the exit code.
 exitval1::
 	@echo $@
-	@$(AWK) -f $(srcdir)/exitval1.awk >_$@ 2>&1; echo EXIT CODE: $$? >>_$@
+	@$(AWK) -f $(srcdir)/exitval1.awk >__$@ 2>&1; echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fsspcoln::
 	@echo $@
-	@$(AWK) -f $(srcdir)/$@.awk 'FS=[ :]+' $(srcdir)/$@.in >_$@
+	@$(AWK) -f $(srcdir)/$@.awk 'FS=[ :]+' $(srcdir)/$@.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsstart1::
 	@echo $@
-	@$(AWK) -f $(srcdir)/$@.awk $(srcdir)/rsstart1.in >_$@
+	@$(AWK) -f $(srcdir)/$@.awk $(srcdir)/rsstart1.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsstart2::
 	@echo $@
-	@$(AWK) -f $(srcdir)/$@.awk $(srcdir)/rsstart1.in >_$@
+	@$(AWK) -f $(srcdir)/$@.awk $(srcdir)/rsstart1.in >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsstart3::
 	@echo $@
-	@head $(srcdir)/rsstart1.in | $(AWK) -f $(srcdir)/rsstart2.awk >_$@
+	@head $(srcdir)/rsstart1.in | $(AWK) -f $(srcdir)/rsstart2.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rtlen::
 	@echo $@
-	@$(srcdir)/$@.sh >_$@ || echo EXIT CODE: $$? >>_$@
+	@$(srcdir)/$@.sh >__$@ || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rtlen01::
 	@echo $@
-	@$(srcdir)/$@.sh >_$@ || echo EXIT CODE: $$? >>_$@
+	@$(srcdir)/$@.sh >__$@ || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rtlenmb::
@@ -1300,110 +1343,130 @@
 
 nondec2::
 	@echo $@
-	@$(AWK) --non-decimal-data -v a=0x1 -f $(srcdir)/$@.awk >_$@
+	@$(AWK) --non-decimal-data -v a=0x1 -f $(srcdir)/$@.awk >__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nofile::
 	@echo $@
-	@$(AWK) '{}' no/such/file >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) '{}' no/such/file >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 binmode1::
 	@echo $@
-	@$(AWK) -v BINMODE=3 'BEGIN { print BINMODE }' >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) -v BINMODE=3 'BEGIN { print BINMODE }' >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 subi18n::
 	@echo $@
-	@GAWKLOCALE=en_US.UTF-8 ; $(AWK) -f $(srcdir)/$@.awk > _$@
+	@GAWKLOCALE=en_US.UTF-8 ; $(AWK) -f $(srcdir)/$@.awk > __$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 concat4::
 	@echo $@
-	@GAWKLOCALE=en_US.UTF-8 ; $(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in > _$@
+	@GAWKLOCALE=en_US.UTF-8 ; $(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in > __$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 devfd1::
 	@echo $@
-	@$(AWK) -f $(srcdir)/$@.awk 4< $(srcdir)/devfd.in1 5< $(srcdir)/devfd.in2 >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) -f $(srcdir)/$@.awk 4< $(srcdir)/devfd.in1 5< $(srcdir)/devfd.in2 >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 # The program text is the '1' which will print each record. How compact can you get?
 devfd2::
 	@echo $@
-	@$(AWK) 1 /dev/fd/4 /dev/fd/5 4< $(srcdir)/devfd.in1 5< $(srcdir)/devfd.in2 >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) 1 /dev/fd/4 /dev/fd/5 4< $(srcdir)/devfd.in1 5< $(srcdir)/devfd.in2 >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mixed1::
 	@echo $@
-	@$(AWK) -f /dev/null --source 'BEGIN {return junk}' >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@$(AWK) -f /dev/null --source 'BEGIN {return junk}' >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mtchi18n::
 	@echo $@
 	@GAWKLOCALE=ru_RU.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 reint2::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) --re-interval -f $@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) --re-interval -f $@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 localenl::
 	@echo $@
-	@$(srcdir)/$@.sh >_$@ 2>/dev/null
+	@$(srcdir)/$@.sh >__$@ 2>/dev/null
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mbprintf1::
 	@echo $@
 	@GAWKLOCALE=en_US.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mbprintf2::
 	@echo $@
 	@GAWKLOCALE=ja_JP.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mbprintf3::
 	@echo $@
 	@GAWKLOCALE=en_US.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mbfw1::
 	@echo $@
 	@GAWKLOCALE=en_US.UTF-8 ; export GAWKLOCALE ; \
-	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	$(AWK) -f $(srcdir)/$@.awk $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >> _$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst6::
 	@echo $@
-	@GAWKLOCALE=C ; $(AWK) -f $(srcdir)/$@.awk > _$@
+	@GAWKLOCALE=C ; $(AWK) -f $(srcdir)/$@.awk > __$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 mbstr1::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 printfbad2: printfbad2.ok
 	@echo $@
-	@$(AWK) --lint -f $(srcdir)/$@.awk $(srcdir)/$@.in 2>&1 | sed 's;\$(srcdir)/;;g' >_$@ || echo EXIT CODE: $$?  >>_$@
+	@$(AWK) --lint -f $(srcdir)/$@.awk $(srcdir)/$@.in 2>&1 | sed 's;\$(srcdir)/;;g' >__$@ || echo EXIT CODE: $$?  >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 beginfile1::
 	@echo $@
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk $(srcdir)/$@.awk . ./no/such/file Makefile  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk $(srcdir)/$@.awk . ./no/such/file Makefile  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 beginfile2:
 	@echo $@
-	@-( cd $(srcdir) && LC_ALL=C AWK="$(abs_builddir)/$(AWKPROG)" $(srcdir)/$@.sh $(srcdir)/$@.in ) > _$@ 2>&1
+	@-( cd $(srcdir) && LC_ALL=C AWK="$(abs_builddir)/$(AWKPROG)" $(srcdir)/$@.sh $(srcdir)/$@.in ) > __$@ 2>&1
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 dumpvars::
@@ -1411,12 +1474,12 @@
 	@AWKPATH=$(srcdir) $(AWK) --dump-variables 1 < $(srcdir)/$@.in >/dev/null 2>&1 || echo EXIT CODE: $$? >>_$@
 	@mv awkvars.out _$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
-
+	
 profile1:
 	@echo $@
 	@$(AWK) --profile=ap-$@.out -f $(srcdir)/xref.awk $(srcdir)/dtdgport.awk > _$@.out1
 	@$(AWK) -f ap-$@.out $(srcdir)/dtdgport.awk > _$@.out2 ; rm ap-$@.out
-	@cmp _$@.out1 _$@.out2 && rm _$@.out[12] || echo EXIT CODE: $$? >>_$@
+	@cmp _$@.out1 _$@.out2 && rm _$@.out[12] || echo EXIT CODE: $$? >>__$@
 
 profile2:
 	@echo $@
@@ -1432,7 +1495,8 @@
 
 posix2008sub:
 	@echo $@
-	@$(AWK) --posix -f $(srcdir)/$@.awk > _$@ 2>&1
+	@$(AWK) --posix -f $(srcdir)/$@.awk > __$@ 2>&1
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 next:
@@ -1442,13 +1506,15 @@
 
 exit:
 	@echo $@
-	@-AWK="$(AWKPROG)" $(srcdir)/$@.sh > _$@ 2>&1
+	@-AWK="$(AWKPROG)" $(srcdir)/$@.sh > __$@ 2>&1
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rri1::
 	@echo $@
 	@[ -z "$$GAWKLOCALE" ] && GAWKLOCALE=en_US.UTF-8; \
-	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 # Targets generated for other tests:
diff -urN old/gawk-4.0.1/test/Maketests new/gawk-4.0.1/test/Maketests
--- old/gawk-4.0.1/test/Maketests	2012-03-28 19:47:54 +0000
+++ new/gawk-4.0.1/test/Maketests	2021-03-29 20:14:11 +0000
@@ -2,1197 +2,1436 @@
 # file Maketests, generated from Makefile.am by the Gentests program
 addcomma:
 	@echo addcomma
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 anchgsub:
 	@echo anchgsub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arrayparm:
 	@echo arrayparm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arrayprm2:
 	@echo arrayprm2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arrayprm3:
 	@echo arrayprm3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arrayref:
 	@echo arrayref
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arrymem1:
 	@echo arrymem1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arryref2:
 	@echo arryref2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arryref3:
 	@echo arryref3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arryref4:
 	@echo arryref4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arryref5:
 	@echo arryref5
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arynasty:
 	@echo arynasty
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm1:
 	@echo aryprm1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm2:
 	@echo aryprm2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm3:
 	@echo aryprm3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm4:
 	@echo aryprm4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm5:
 	@echo aryprm5
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm6:
 	@echo aryprm6
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm7:
 	@echo aryprm7
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aryprm8:
 	@echo aryprm8
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arysubnm:
 	@echo arysubnm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 asgext:
 	@echo asgext
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 back89:
 	@echo back89
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 backgsub:
 	@echo backgsub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 childin:
 	@echo childin
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 closebad:
 	@echo closebad
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 clsflnam:
 	@echo clsflnam
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 compare2:
 	@echo compare2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 concat1:
 	@echo concat1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 concat2:
 	@echo concat2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 concat3:
 	@echo concat3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 convfmt:
 	@echo convfmt
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 datanonl:
 	@echo datanonl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 defref:
 	@echo defref
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 delargv:
 	@echo delargv
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 delarpm2:
 	@echo delarpm2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 delarprm:
 	@echo delarprm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 delfunc:
 	@echo delfunc
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 dfastress:
 	@echo dfastress
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 dynlj:
 	@echo dynlj
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 eofsplit:
 	@echo eofsplit
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 exitval2:
 	@echo exitval2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fcall_exit:
 	@echo fcall_exit
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fcall_exit2:
 	@echo fcall_exit2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fldchg:
 	@echo fldchg
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fldchgnf:
 	@echo fldchgnf
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnamedat:
 	@echo fnamedat
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnarray:
 	@echo fnarray
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnarray2:
 	@echo fnarray2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnaryscl:
 	@echo fnaryscl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnasgnm:
 	@echo fnasgnm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnmisc:
 	@echo fnmisc
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fordel:
 	@echo fordel
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 forref:
 	@echo forref
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 forsimp:
 	@echo forsimp
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fsbs:
 	@echo fsbs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fsrs:
 	@echo fsrs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fstabplus:
 	@echo fstabplus
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 funsemnl:
 	@echo funsemnl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 funsmnam:
 	@echo funsmnam
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 funstack:
 	@echo funstack
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getline:
 	@echo getline
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getline3:
 	@echo getline3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getline4:
 	@echo getline4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getnr2tb:
 	@echo getnr2tb
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getnr2tm:
 	@echo getnr2tm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubasgn:
 	@echo gsubasgn
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtest:
 	@echo gsubtest
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst2:
 	@echo gsubtst2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst4:
 	@echo gsubtst4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst5:
 	@echo gsubtst5
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst7:
 	@echo gsubtst7
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gsubtst8:
 	@echo gsubtst8
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 hex:
 	@echo hex
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 hsprint:
 	@echo hsprint
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 inputred:
 	@echo inputred
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 intest:
 	@echo intest
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 intprec:
 	@echo intprec
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 iobug1:
 	@echo iobug1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 leadnl:
 	@echo leadnl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 longsub:
 	@echo longsub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 longwrds:
 	@echo longwrds
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 manglprm:
 	@echo manglprm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 math:
 	@echo math
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 membug1:
 	@echo membug1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 minusstr:
 	@echo minusstr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nasty:
 	@echo nasty
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nasty2:
 	@echo nasty2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 negexp:
 	@echo negexp
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 negrange:
 	@echo negrange
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nested:
 	@echo nested
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nfldstr:
 	@echo nfldstr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nfneg:
 	@echo nfneg
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nfset:
 	@echo nfset
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nlfldsep:
 	@echo nlfldsep
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nlinstr:
 	@echo nlinstr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nlstrina:
 	@echo nlstrina
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 noeffect:
 	@echo noeffect
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nofmtch:
 	@echo nofmtch
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 noloop1:
 	@echo noloop1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 noloop2:
 	@echo noloop2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 noparms:
 	@echo noparms
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nulrsend:
 	@echo nulrsend
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 numindex:
 	@echo numindex
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 numsubstr:
 	@echo numsubstr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 octsub:
 	@echo octsub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ofmt:
 	@echo ofmt
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ofmta:
 	@echo ofmta
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ofmtbig:
 	@echo ofmtbig
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ofmtfidl:
 	@echo ofmtfidl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ofmts:
 	@echo ofmts
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 onlynl:
 	@echo onlynl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 opasnidx:
 	@echo opasnidx
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 opasnslf:
 	@echo opasnslf
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 paramdup:
 	@echo paramdup
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 paramres:
 	@echo paramres
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 paramtyp:
 	@echo paramtyp
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 parse1:
 	@echo parse1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 parsefld:
 	@echo parsefld
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 parseme:
 	@echo parseme
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 pcntplus:
 	@echo pcntplus
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prdupval:
 	@echo prdupval
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prec:
 	@echo prec
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 printf1:
 	@echo printf1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prmarscl:
 	@echo prmarscl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prmreuse:
 	@echo prmreuse
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prt1eval:
 	@echo prt1eval
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 prtoeval:
 	@echo prtoeval
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rand:
 	@echo rand
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 range1:
 	@echo range1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rebt8b1:
 	@echo rebt8b1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 regeq:
 	@echo regeq
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 regrange:
 	@echo regrange
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 reindops:
 	@echo reindops
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 reparse:
 	@echo reparse
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 resplit:
 	@echo resplit
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rs:
 	@echo rs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rsnul1nl:
 	@echo rsnul1nl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest1:
 	@echo rstest1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest2:
 	@echo rstest2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest3:
 	@echo rstest3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest4:
 	@echo rstest4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest5:
 	@echo rstest5
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rswhite:
 	@echo rswhite
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 scalar:
 	@echo scalar
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sclforin:
 	@echo sclforin
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sclifin:
 	@echo sclifin
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sortempty:
 	@echo sortempty
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitargv:
 	@echo splitargv
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitarr:
 	@echo splitarr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitdef:
 	@echo splitdef
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitvar:
 	@echo splitvar
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitwht:
 	@echo splitwht
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 strcat1:
 	@echo strcat1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 strnum1:
 	@echo strnum1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 strtod:
 	@echo strtod
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 subsepnm:
 	@echo subsepnm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 subslash:
 	@echo subslash
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 substr:
 	@echo substr
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 swaplns:
 	@echo swaplns
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 synerr1:
 	@echo synerr1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 synerr2:
 	@echo synerr2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uninit2:
 	@echo uninit2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uninit3:
 	@echo uninit3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uninit4:
 	@echo uninit4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uninit5:
 	@echo uninit5
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uninitialized:
 	@echo uninitialized
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 unterm:
 	@echo unterm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 uparrfs:
 	@echo uparrfs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 wjposer1:
 	@echo wjposer1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 zero2:
 	@echo zero2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 zeroe0:
 	@echo zeroe0
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 zeroflag:
 	@echo zeroflag
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getlnhd:
 	@echo getlnhd
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aadelete1:
 	@echo aadelete1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aadelete2:
 	@echo aadelete2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aarray1:
 	@echo aarray1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aasort:
 	@echo aasort
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 aasorti:
 	@echo aasorti
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 arraysort:
 	@echo arraysort
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 backw:
 	@echo backw
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 clos1way:
 	@echo clos1way
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 delsub:
 	@echo delsub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fieldwdth:
 	@echo fieldwdth
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fpat1:
 	@echo fpat1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fpat2:
 	@echo fpat2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fpat3:
 	@echo fpat3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fpatnull:
 	@echo fpatnull
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fsfwfs:
 	@echo fsfwfs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 funlen:
 	@echo funlen
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fwtest:
 	@echo fwtest
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fwtest2:
 	@echo fwtest2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fwtest3:
 	@echo fwtest3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gensub:
 	@echo gensub
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gensub2:
 	@echo gensub2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 getlndir:
 	@echo getlndir
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gnuops2:
 	@echo gnuops2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gnuops3:
 	@echo gnuops3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 gnureops:
 	@echo gnureops
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 icasefs:
 	@echo icasefs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 icasers:
 	@echo icasers
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 igncdym:
 	@echo igncdym
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 igncfs:
 	@echo igncfs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 ignrcase:
 	@echo ignrcase
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 indirectcall:
 	@echo indirectcall
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 lint:
 	@echo lint
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 lintold:
 	@echo lintold
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint-old < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint-old < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 lintwarn:
 	@echo lintwarn
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 match1:
 	@echo match1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 match2:
 	@echo match2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 match3:
 	@echo match3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nastyparm:
 	@echo nastyparm
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 nondec:
 	@echo nondec
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 patsplit:
 	@echo patsplit
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 posix:
 	@echo posix
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 printfbad1:
 	@echo printfbad1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 printfbad3:
 	@echo printfbad3
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 procinfs:
 	@echo procinfs
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 pty1:
 	@echo pty1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rebuf:
 	@echo rebuf
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 regx8bit:
 	@echo regx8bit
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rstest6:
 	@echo rstest6
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 shadow:
 	@echo shadow
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  --lint >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sortfor:
 	@echo sortfor
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sortu:
 	@echo sortu
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 splitarg4:
 	@echo splitarg4
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 strtonum:
 	@echo strtonum
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 switch2:
 	@echo switch2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 double1:
 	@echo double1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 double2:
 	@echo double2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 intformat:
 	@echo intformat
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 asort:
 	@echo asort
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 asorti:
 	@echo asorti
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fmttest:
 	@echo fmttest
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnarydel:
 	@echo fnarydel
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 fnparydl:
 	@echo fnparydl
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 rebt8b2:
 	@echo rebt8b2
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sort1:
 	@echo sort1
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 sprintfc:
 	@echo sprintfc
-	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
+	@AWKPATH=$(srcdir) $(AWK) -f $@.awk  < $(srcdir)/$@.in >__$@ 2>&1 || echo EXIT CODE: $$? >>__$@
+	@sed "s|awk$(EXEEXT)|awk|g" < __$@ > _$@ && rm -f __$@
 	@-$(CMP) $(srcdir)/$@.ok _$@ && rm -f _$@
 
 # end of file Maketests
